Korath campaign: first contact, establish lore, side story, and Korath-protected smuggler's haven (#85)
endless-sky/Endless-Sky-Delta: data/korath/korath missions.txt

			`	You approach one of the Quarg in the spaceport and request assistance with the data chip. It stops and acknowledges you, but doesn't speak. It pauses for a few moments after you make your request, but then holds out its hand. Taking the data chip, it slots it into a communicator of some sort, slowly nods to itself, and hands it back to you.`
			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
			`	You carefully store the chip in a safe place.`

> 

Reviewer: ```suggestion

```

			label parting
			`	"Kai," says the Korath, and they stretch out both arms toward you, palms out. You make the same gesture and touch their hands, palm to palm. You're not sure if that is the right response, but the Korath seem satisfied, saying, "Nik lak'mal." They get back in their fighters, take off, and hover nearby.`
				decline

mission "Korath [F01]: A Fishy Deal"
	landing
	name "A fishy proposition"
	description "Transport <cargo> to <planet> by <date>. After a return trip, you'll be paid 140,000 credits. Don't get caught."
	cargo "live fish" 15
	# The mission is illegal because of the broader trade agreement. The player is supposed to be left with the impression that the Korath are sending back more dangerous things than waste processing systems. The Syndicate (and by extension, the Republic) is aware of this, and fines pilots involved.
	illegal 50000
	deadline 4
	clearance "The authorities do not respond, but a concealed hangar bay door opens. The meaning is clear: land here."
	source "New Tortuga"
	destination "La'ab Bamen"
	on offer
		event "know about Korath fish jobs"
		conversation

> 			`Flying through the atmosphere of New Tortuga, you pass over a typhoon, overfly a tsunami, and take a short detour around a supercell thunderstorm. After a half hour of delicate flying, you approach doors more than a hundred meters tall. As they open, you can see they are ten meters thick, made of an unfamiliar brown material. Two layers of shields shut down in anticipation of your entry, each strong enough to stop a Navy Carrier. To the left and right, there are turrets with heat signatures higher than a Breeder Reactor. Whoever built this place values their safety... and has the funds and technology to secure it.`

Reviewer: Adaptive knowledge of heat.
```suggestion
			`Flying through the atmosphere of New Tortuga, you pass over a typhoon, overfly a tsunami, and take a short detour around a supercell thunderstorm. After a half hour of delicate flying, you approach doors more than a hundred meters tall. As they open, you can see they are ten meters thick, made of an unfamiliar brown material. Two layers of shields shut down in anticipation of your entry, each strong enough to stop a Navy Carrier. To the left and right, there are turrets with heat signatures higher than a Breeder Reactor. Whoever built this place values their safety... and has the funds and technology to secure it.`
				to display
					has "outfit (flagship installed): Thermal Scanner"
			`Flying through the atmosphere of New Tortuga, you pass over a typhoon, overfly a tsunami, and take a short detour around a supercell thunderstorm. After a half hour of delicate flying, you approach doors more than a hundred meters tall. As they open, you can see they are ten meters thick, made of an unfamiliar brown material. Two layers of shields shut down in anticipation of your entry, each strong enough to stop a Navy Carrier. To the left and right, there are huge turrets with heat emanating outwards. Whoever built this place values their safety... and has the funds and technology to secure it.`
				to display
					not "outfit (flagship installed): Thermal Scanner"
```

Author: No. This is too much duplicated text for such a minor thing. Also, the idea that a space ship who needs to know where to land wouldn't have infrared scanners is so unrealistic you're stretching suspension of disbelief to the breaking point.

					decline
					to display
						or
							has "Korath [F01]: A Fishy Deal: ship"
							has "Korath [F01]: A Fishy Deal: down"
							has "Korath [F01]: A Fishy Deal: mission"
				`	(Ignore her and go about your business.)`
					goto "decline mission"
					to display
						not "Korath [F01]: A Fishy Deal: ship"
						not "Korath [F01]: A Fishy Deal: down"
						not "Korath [F01]: A Fishy Deal: mission"
			label ship
			action
				set "Korath [F01]: A Fishy Deal: ship"
				log "Minor People" "Jenny Jetmaster" "Latest in a family line that has operated and maintained a replica of an ancient jet aircraft and made money on airshows. She is now on the run from the Syndicate for rescuing her persecuted uncle from Tinker."
			`	You see her expression brighten as she walks towards you. "It's an heirloom," she says proudly, gesturing widely at the ship, "which has been in the family for centuries. My distant ancestor got a military surplus cargo plane and refurbished it. They used it as a personal craft. Each generation added to it; every piece has been repaired or replaced over the last thousand years. It's more of a replica now.`

> 			`	"The whole thing has been modernized - about a dozen times. There's a hyperdrive built into the hull itself, and those ion engines aren't of Syndicate design. They're custom-built, with ion propulsion, steering, and atmospheric anti-grav, in one assembly. The design was once called a Jetmaster, and my family have taken the Jetmaster name as our own.`

Reviewer: Here is where.

Author: Ah, that's easy. I'll take out "anti-grav"

> They're custom-built, with ion propulsion, steering, and atmospheric, all in one assembly.

By the way, the ship is a McDonnell Douglas/Boeing C-17 Globemaster III. Being that specific in the story would cause problems. Hence, "Jetmaster."

Reviewer 2: Sorry, but I was actually good with this reference. As previously mentioned, anti-grav as a basic principle is moderately established as being something routinely used to get spacecraft into space without needing boosters or whatnot. I think it's probably fairly universal, even. So this reference here is appropriate to humanity's tech level. Namely, being able to build an object that reduces the impact of gravity on the item itself and that which is attached to it. 

It's the "projecting an artificial gravity field" that I don't think is appropriate for all the possible ships the player could be flying when they encouter the series of statements.

Author: I did a recursive search of all txt files in the repository for the regular expression `project.*artif.*grav`, caselessly, and I don't see that anywhere. Can you tell me where it is?

EDIT: Zitchas's later comment told me where it is.

			`	Certainly, a second landing attempt will get you through the atmosphere. Next, you'll have to scout for a safe place to land on this world of water and storms.`
				launch

			label "second attempt"
			scene scene/volcanic-eruption-at-night
			action
				log "Was able to pass through the atmosphere of Rabasot Pa'ah despite the unusual atmospheric event. A volcanic eruption nearly hit the ship, perhaps caused by landing in the wrong area. There is a large island that might be suitable for landing. Another attempt could succeed."
			`In this trip, your ship's descent stirred up awe-inspiring storm clouds, but traveling through them was easy. Your shield plow technique took you down to the lower troposphere, where the storms cleared enough for you to see. The original island chain was too far away, so you set course for another one nearby.`

> 			`	Aiming for the hardest-looking piece of ground your sensors can find, you touch down. As this planet has twice Earth's gravity, you extend your ship's artificial gravity field to the surface. Donning a space suit for safety's sake, you walk out to see nature taking control once again: the massive volcano in front of you starts to erupt.`

Reviewer 2: ```suggestion
			`	Aiming for the hardest-looking piece of ground your sensors can find, you touch down. Noting the extensive radiation, you extend your ship's shields to the surface. Donning a space suit for safety's sake, you walk out to see nature taking control once again: the massive volcano in front of you starts to erupt.`
```

This is the artificial gravity field reference I was referring to. 

I'd be 100% OK with a a *Korath* ship modulating a gravitational modifying field like this. (Or the Quarg, Drak, Pug, or similarly highly advanced factions with a demonstrated history of not just antigrav, but actual grav generation, projection, and modification.) But on a human ship? I don't like thinking of human ships being that advanced. And I think there's a faction that is intended to be a bit less advanced than humanity out there that the player could theoretically be using one of their ships to encounter this.

Author: I see what you mean.

I was referring to the simple gravity deck plating, not anything fancy. I'll get rid of that.

				set "Korath [F01]: A Fishy Deal: ship"
				log "Minor People" "Jenny Jetmaster" "Latest in a family line that has operated and maintained a replica of an ancient jet aircraft and made money on airshows. She is now on the run from the Syndicate for rescuing her persecuted uncle from Tinker."
			`	You see her expression brighten as she walks towards you. "It's an heirloom," she says proudly, gesturing widely at the ship, "which has been in the family for centuries. My distant ancestor got a military surplus cargo plane and refurbished it. They used it as a personal craft. Each generation added to it; every piece has been repaired or replaced over the last thousand years. It's more of a replica now.`

> 			`	"The whole thing has been modernized - about a dozen times. There's a hyperdrive built into the hull itself, and those ion engines aren't of Syndicate design. They're custom-built, with ion propulsion, steering, and atmospheric, all in one assembly. The design was once called a Jetmaster, and my family have taken the Jetmaster name as our own.`

Reviewer 2: ```suggestion
			`	"The whole thing has been modernized - about a dozen times. There's a hyperdrive built into the hull itself, and those ion engines aren't of Syndicate design. They're custom-built, with ion propulsion, steering, and atmospheric anti-grav, all in one assembly. The design was once called a Jetmaster, and my family have taken the Jetmaster name as our own.`
```
